ext/bg/lang/deinflect.json

```diff
@@ -784,6 +784,166 @@
             "rulesOut": [
                 "adj-i"
             ]
+        },
+        {
+            "kanaIn": "のたもうたら",
+            "kanaOut": "のたまう",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "いったら",
+            "kanaOut": "いく",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "おうたら",
+            "kanaOut": "おう",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "こうたら",
+            "kanaOut": "こう",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "そうたら",
+            "kanaOut": "そう",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "とうたら",
+            "kanaOut": "とう",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "行ったら",
+            "kanaOut": "行く",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "逝ったら",
+            "kanaOut": "逝く",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "往ったら",
+            "kanaOut": "往く",
+            "rulesIn": [],
+            "rulesOut": [
+                "v5"
+            ]
+        },
+        {
+            "kanaIn": "逝ったり",
```

Comment:
I wonder what's with the 〜たり entries in the latter half of the `-tara` block and similarly 〜たら in the `-tari` block below. Otherwise the new conjugations look fine to me